test/functional/specs/Privacy/IAB/C224677.js

```diff
@@ -67,11 +65,17 @@ test("Test C224677: Call setConsent when purpose 10 is FALSE", async () => {
   const identityHandle = response.getPayloadsByType("identity:result");
   await t.expect(identityHandle.length).eql(2);
 
-  // Event calls going forward should be Opted-Out because AAM opts out consents with no purpose 10.
-  const errorMessage = await getErrorMessageFromSendEvent();
+  // 3. Event calls going forward should be Opted-Out because AAM opts out consents with no purpose 10.
```

Comment:
Grammatically, "Opted-Out" here should be "opted out" (it's the past tense of a transitive verb).